backend/lucene/src/main/java/org/hibernate/search/backend/lucene/impl/LuceneBackendImpl.java

```diff
@@ -19,14 +19,14 @@
 import org.hibernate.search.backend.lucene.multitenancy.impl.MultiTenancyStrategy;
 import org.hibernate.search.backend.lucene.orchestration.impl.LuceneSyncWorkOrchestratorImpl;
 import org.hibernate.search.backend.lucene.resources.impl.BackendThreads;
-import org.hibernate.search.backend.lucene.common.timing.impl.TimingSource;
```

Comment:
You should probably merge the commit `HSEARCH-3787 Remove lucene-level TimingSource ` with the previous one. Otherwise you're missing out on git's ability to track renamed files and to provide a commit history for a particular file even beyond its renaming.